pycg/processing/postprocessor.py

```diff
@@ -50,8 +64,8 @@ def visit_Lambda(self, node):
         super().visit_Lambda(node, lambda_name)
         logger.debug("Exit PreProcessor.visit_Lambda")
 
-    def visit_Call(self, node):
-        logger.debug("In PreProcessor.visit_Call")
+    def visit_Call(self, node: ast.Call):
+        # logger.debug("In PreProcessor.visit_Call")
```

Comment:
I tried not to include many unrelated typographical changes in this PR, but I cleaned up my work by running ruff format and the diffs were stuck together in a few of these.